microsoft-edge/extensions-chromium/publish/api/using-addons-publish-api.md

```diff
@@ -0,0 +1,222 @@
+| _operation ID_ | The ID of a REST operation. |
+
+
+<!-- ====================================================================== -->
+## Before you begin
+
+To use the Microsoft Edge Add-ons Publish API, you need to enable the API for your project in the Microsoft Partner Center.
+
+1. Visit Microsoft Partner Center and login to the account from which you have already published an add-on.
+
+1. Under the **Microsoft Edge** program, a new page for **Publish APIs** appears.
+
+1. Enable the APIs.<!-- what UI action to do that?--> This step may take a few minutes.  After the APIs are enabled, the **Client ID**, **Client Secret**, and **Auth Token URL** are displayed on this page.
+
+1. Note the **ClientID**, **Client Secret** and the **Auth Token URL**. You will use them in the next step to get an access token.
+
+
+<!-- ====================================================================== -->
+## Retrieving the access token
+
+After you've acquired the necessary authorization for your application, proceed with acquiring access tokens for APIs.
+
+1. To get a token using the client credentials grant, send a POST request to the Auth Token URL.  The tenant information is available in the URL that you received in the **Before you begin** steps above.
+
+```console
+Endpoint: https://login.microsoftonline.com/{tenant}/oauth2/v2.0/token
+Type: POST
+Header Parameters: Content-Type: application/x-www-form-urlencoded
+```
+
+### Sample request
+
+```console
+> curl \
+-X POST \
+-H "Content-Type: application/x-www-form-urlencoded" \
+-d "client_id={$Client_ID}" \
+-d "scope=https://addons.edge.microsoft.com/.default" \
+-d "client_secret={$Client_Secret}" \
+-d "grant_type=client_credentials" \
+-v \
+https://login.microsoftonline.com/{tenant}/oauth2/v2.0/token
+```
+
+### Sample response
+
+```json
+{
+  "token_type": "Bearer",
+  "expires_in": 3599,
+  "access_token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6Ik1uQ19WWmNBVGZNNXBP..."
+}
+```
+
+For more information, navigate to [OAuth 2.0 client credentials flow on the Microsoft identity platform][AzureOAuthGetToken].
+
+
+<!-- ====================================================================== -->
+## Using the Publish API
+
+Once you have an access token, your app can use the Microsoft Edge Add-ons Publish API.  There are endpoints for getting a list of products, updating a product, and publishing a product.<!-- correct list of actions? -->
+
+> [!NOTE]
+> Currently, there is no API for creating a new product or updating a product's metadata, such as a description.  You must complete these tasks manually in Microsoft Partner Center.
+
+The examples below use the domain `https://addons.edge.microsoft.com/api`, which is a placeholder and needs to be replaced by another example, given that this API is not live yet.
+
+
+<!-- ====================================================================== -->
+## Getting the list of products
+
+<!-- no section in Part 1 of spec -->
```

Comment:
Created this section to give consistent parallel w/ API Ref article.  Add info here like the following sections?

Reply:
Not needed, as the Products API is an optional API. It has been referenced in the How to Get product ID section. May be it needs a hyperlink there.